### DISCUSSION TITLE
按←后退5秒后又会前进5秒，按→会前进10秒

### DISCUSSION THREAD

**Discussion by Felrix** [hooke007/mpv_PlayKit#76] [Q&A 问答] [answered]
mp4文件
按一次←后退5秒后又会前进5秒，按两次←后退10秒后又会前进10秒，但是按三次←后退15秒只前进5秒；
按→会前进10秒
版本mpv-lazy-2021ex

**Comment by hooke007** [comment, marked as answer]
更换不同视频文件测试
请检查是否该视频自身的编码问题

**Reply by Felrix** [reply]
测试了4个mp4，1个flv，其中3个来自同一源的mp4出现上述情况，其他视频均正常。
出问题的视频编码标准为H.264
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/66423087/147407491-3990203e-fefb-4bbf-bccb-102206922ff3.png">
如何解决这个问题（期末就等着这个播放器救命了

**Reply by hooke007** [reply]
精确搜索还能用的话就用精确
`seek  5 exact`

**Reply by Felrix** [reply]
试了下，还是不行

**Reply by hooke007** [reply]
mpegts好像没有索引，除了重编码应该无解了

**Reply by Felrix** [reply]
学习了一下怎么编码，果然重新编码就好了😂
